src/hotspot/share/gc/g1/g1BarrierSet.hpp

```diff
@@ -49,9 +49,10 @@ class G1BarrierSet: public CardTableBarrierSet {
 
   void invalidate(JavaThread* thread, MemRegion mr);
 
+  ~G1BarrierSet() = default;
```

Comment:
Why is this being privatized?

Reply:
To minimize the public API; it doesn't need to be public.

Reply:
It should be public - this is a directly instantiated class.  Yes, I know we don't currently call it.